firefox-ios/Shared/Supporting Files/be.lproj/Settings.strings

```diff
@@ -118,12 +118,6 @@
 /* In the search page of the Settings menu, the title for the link to the SUMO Page about Firefox Suggest. */
 "Settings.Search.Suggest.LearnAboutSuggestions.v124" = "Даведайцеся больш пра Firefox Suggest";
 
-/* In the Search page of the Settings menu, the description for the setting to enable Suggestions from Firefox. */
-"Settings.Search.Suggest.ShowNonSponsoredSuggestions.Description.v124" = "Атрымлівайце прапановы з інтэрнэту, звязаныя з вашым пошукам";
-
-/* In the Search page of the Settings menu, the title for setting to enable Suggestions from Firefox. Placeholder is for the app name - Firefox. */
-"Settings.Search.Suggest.ShowNonSponsoredSuggestions.Title.v124" = "Прапановы ад %@";
-
```

Comment:
@adudenamedruby do you know why those strings are getting removed here? Seems like they shouldn't? I see them in the `main` Strings file

Reply:
If we re-run this, it should be fine now. :)